x-pack/legacy/plugins/uptime/public/components/functional/charts/__tests__/donut_chart.test.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { DonutChart } from '../donut_chart';
+import { renderWithIntl, shallowWithIntl } from 'test_utils/enzyme_helpers';
+import React from 'react';
+
+describe('DonutChart component', () => {
+  it('passes correct props without errors for valid props', () => {
+    const props = {
+      down: 32,
+      up: 95,
+      height: 125,
+      width: 125,
+    };
+
+    const wrapper = shallowWithIntl(<DonutChart {...props} />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('renders a donut chart', () => {
+    const props = {
+      down: 32,
+      up: 95,
+      height: 125,
+      width: 125,
+    };
+
+    const wrapper = renderWithIntl(<DonutChart {...props} />);
+    expect(wrapper).toMatchSnapshot();
+  });
```

Comment:
Both tests seems to be doing the same thing.

Reply:
One's using `shallow` (checking the props are going to the correct spots) and the other is `rendering` to make sure the d3 code is producing reliable svg elements. We can probably extract the props instantiation to a `beforeEach` setup.